AppliancesModel/Contracts/IAppliances.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
can not build solution
verify please

Reply:
Tried to clone repository and loaded only this branch. Solution builded success.